### ISSUE TITLE
Proposal: add repository security advisory

### ISSUE THREAD

**Issue by bjohansebas** [expressjs/discussions#277]
GitHub has a feature that allows creating a draft security advisory to privately discuss and fix a security vulnerability, similar to how issue templates work.

For example, Next.js has this feature enabled:
![image](https://github.com/user-attachments/assets/946ca24b-ebe6-4b01-b37c-40d12431b044)

I believe this would improve the process of reporting potential vulnerabilities in Express and its packages.

ref: 
- https://docs.github.com/en/code-security/security-advisories/guidance-on-reporting-and-writing-information-about-vulnerabilities/privately-reporting-a-security-vulnerability
- https://docs.github.com/en/code-security/security-advisories/working-with-repository-security-advisories/creating-a-repository-security-advisory

**Comment by UlisesGascon** [comment]
Yep! We will add this to all the repos at some point (hope soon). I will transfer the issue to the Security-wg for execution :+1:
